core/drivers/crypto/hisilicon/crypto.mk

```diff
@@ -0,0 +1,11 @@
+ifeq ($(CFG_HISILICON_ACC_V3), y)
+$(call force,CFG_CRYPTO_DRIVER,y)
+CFG_CRYPTO_DRIVER_DEBUG ?= 0
+
+$(call force, CFG_CRYPTO_DRV_ACIPHER,y,Mandated by CFG_HISILICON_CRYPTO_DRIVER)
```

Comment:
Prefer "Required" over "Mandated", here and at line 8.

Reply:
what is the diffrence between "Required" and "Mandated"

Reply:
Get it.Thanks!